- player/main: log libplacebo version if built with it
  0f0520aeca367613d313dbf2e936ea2cacdb35c9

Index: player/main.c
--- player/main.c.orig
+++ player/main.c
@@ -25,6 +25,11 @@
 #include <locale.h>
 
 #include "config.h"
+
+#if HAVE_LIBPLACEBO
+#include <libplacebo/config.h>
+#endif
+
 #include "mpv_talloc.h"
 
 #include "misc/dispatch.h"
@@ -144,6 +149,9 @@ void mp_print_version(struct mp_log *log, int always)
     int v = always ? MSGL_INFO : MSGL_V;
     mp_msg(log, v, "%s %s\n built on %s\n",
            mpv_version, mpv_copyright, mpv_builddate);
+#if HAVE_LIBPLACEBO
+    mp_msg(log, v, "libplacebo version: %s\n", PL_VERSION);
+#endif
     check_library_versions(log, v);
     mp_msg(log, v, "\n");
     // Only in verbose mode.
